driver.go

```diff
@@ -1,3 +1,4 @@
+// Package exasol implements a driver for the Exasol database.
```

Comment:
Should we also add a copyright here or add usage example like https://github.com/go-sql-driver/mysql/blob/master/driver.go mysql is doing it?

Reply:
👍 ✅